app/views/hyrax/base/_form_dates.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="date_wrapper">
+  <div class="row"> 
+    <legend>Dates</legend>
+    <div class="warning-anchor"></div> 
+    <div class="col-sm-6">
+      <select name="new_date_type" id="new_date_type" class="form-control">
+        <option value="Select a date type">Select a date type</option>
+        <option value="accepted">Accepted</option>
```

Comment:
we should probably try to make these options supplied by the form, so that the value and text are a bit more dynamic. I'm thinking the first option "Select a date type" is good, but then mapping the date_terms..maybe something along these lines:

`f.object.date_terms.map { |t| "<option value='#{t.to_s}'>#{t.humanize}</option>" }`

I'm not 100% sure on the syntax.. but the goal would be so that the value of each option has a guarantee to match one of the date_term partials produced in the `render_edit_field_partial....` below. The JS that would drive this feature would detect the `change` event on this select, grab the value of the selected option and then find the related date_term partial below and make it visible.